dev_tools/locustfile.py

```diff
@@ -0,0 +1,36 @@
+"""Locust (https://docs.locust.io/en/stable/what-is-locust.html) is a load testing tool.
+To run Locust, set the SUPERSTAQ_API_KEY environment variable and execute
+"locust --config=locust.conf" in the "dev_tools" directory. Then
+navigate to http://0.0.0.0:8089/, insert the number of users and spawn rate,
+then click the "Start swarming" button.
+"""
+
+from __future__ import annotations
+
+import logging
+
+import cirq_superstaq
```

Comment:
```suggestion
import cirq_superstaq as css
```
^ consistency with the rest of codebase (per vote last summer)

Reply:
Also fixed https://github.com/SupertechLabs/client-superstaq/pull/518